modules/.gitignore

```diff
@@ -15,6 +15,8 @@ lib/
 liferay-hook.xml.processed
 node_modules/
 npm-debug.log
+!src/main/resources/META-INF/resources/lib
```

Comment:
in bundler 2.x, the angular templates have stuff in that directory

Reply:
Right, but if you didn't have that line in gitigore would it get ignored?  It seems that is unncessary since normally that folder would not be ignored right?

Reply:
on https://github.com/petershin/liferay-portal/blob/c94e5b55a7f00e5d1673df9000b2ab13b3ae9f45/modules/.gitignore#L14
`lib/` is ignored, so that's why i needed this

Reply:
should we rename the folder to js?